wvd-templates/Create and provision WVD host pool/mainTemplate.json

```diff
@@ -275,6 +282,9 @@
                     "rdshVmSize": {
                         "value": "[parameters('rdshVmSize')]"
                     },
+                    "enableAcceleratedNetworking": {
```

Comment:
There is kind of limited support for Accelerated Networking (https://docs.microsoft.com/en-us/azure/virtual-network/create-vm-accelerated-networking-powershell#limitations-and-constraints), so I don't think we should default to the parameter. I think that we should reference a variable and do value checking there. Currently, it's limited to Azure Gallery images, and from there it's only WS 2012R2 and WS 2016.

Reply:
The parameter defaults to "false' already but it works with Windows Server 2019. It also applies to custom images as well, the portal disables the option because there is no way for ARM to identify at the deployment time if the custom image supports it, I had discussions around this last week with the Linux PM since I was facing this issue of the AN button being disabled but from Azure CLI, Powershell and Template we can apply it.